mllib/src/test/scala/org/apache/spark/mllib/clustering/LDASuite.scala

```diff
@@ -131,8 +131,8 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext {
 
   test("setter alias") {
     val lda = new LDA().setAlpha(2.0).setBeta(3.0)
-    assert(lda.getAlpha === 2.0)
-    assert(lda.getDocConcentration === 2.0)
+    assert(lda.getAlpha.toArray.forall(_ === 2.0))
```

Comment:
Order of setting k and alpha no longer matters (see discussion L123).

Setter taking vector is tested in `test("OnlineLDAOptimizer with asymmetric prior")`.

Will add test for sad path.